### PR TITLE
Fix/doc improvements

### PR DESCRIPTION
**Pull request by shuldt**
## Changes in this PR

Adds a doc to make uploading and downloading files to the remotes easier in future. Also adds some details to existing documentation for uploading forecast files to avoid time loss to errors in future. 

## Next steps

There are no next steps, this PR does not affect functionality.

### CODE REVIEW

**Review by pezholio — APPROVED** [review]
LGTM 👍

File: doc/uploading-and-dowloading-files.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+## Uploading files
+
+- to copy a file to the required remote environment:
+
+```
+cat RELATIVE-PATH/FILENAME.csv | cf ssh beis-roda-ENVIRONMENT -c "cat > FILENAME.csv"
+```
+
+Note: `cat` cannot create a directory. This command will fail if a path is passed to `cat >` . This also means that the copied file will be found in the **root** of the remote project; you will need to move the file to the required directory after copying. 
+
```
Comment (edavey) [inline doc/uploading-and-dowloading-files.md:12]: Perhaps excessively nit-picky, but I don't think this is quite accurate -- I think the uploaded file ends up in `/root` which is the home directory (aka `~/`) of the logged-in user `root`:

```
$ cat local_foo.txt | cf ssh beis-roda-staging -c "cat > FILENAME.csv"
$ cf ssh beis-roda-staging                      
root@7f3df505-fd30-4569-50f2-a078:/app# cd ~
root@7f3df505-fd30-4569-50f2-a078:~# ls
FILENAME.csv
root@7f3df505-fd30-4569-50f2-a078:~# pwd
/root
root@7f3df505-fd30-4569-50f2-a078:~# cd /
root@7f3df505-fd30-4569-50f2-a078:/# ls
app  boot  dev			 etc   lib    media  opt   root  sbin  sys  usr
bin  deps  docker-entrypoint.sh  home  lib64  mnt    proc  run	 srv   tmp  var
root@7f3df505-fd30-4569-50f2-a078:/# pwd
/
```

Feel free to disregard if you want!

**Comment by edavey** [comment]
I think you could squash `f6db82f` and `707ac2e` -- as the latter is a correction? i.e. the history will be clearer if we remove the initial mistake?

**Review by edavey — APPROVED** [review]
Nice improvements!

I've made a couple of tiny suggestions...

File: doc/uploading-and-dowloading-files.md (hunk 1 above)
Comment (shuldt) [reply doc/uploading-and-dowloading-files.md:12]: No I think you're absolutely right and I will amend it

File: doc/importing-new-delivery-partner-data.md [hunk 2]
```diff
@@ -223,9 +223,11 @@ contains anything earlier, delete those columns.
 - copy the file over to prod
 
 ```
-cat FILENAME.csv | cf ssh beis-roda-prod -c "cat > FILENAME.csv"
+cat RELATIVE-PATH/FILENAME.csv | cf ssh beis-roda-prod -c "cat > FILENAME.csv"
 ```
 
+Note: `cat` cannot create a directory. This command will fail if a path is passed to `cat >` . This also means that the copied file will be found in the **root** of the remote project; you will need to move the file to the required directory after copying. 
+
```
Comment (edavey) [inline doc/importing-new-delivery-partner-data.md:230]: is it also worth including the revised version here?

**Pull request merged**